setup.py

```diff
@@ -12,11 +12,13 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+import re
+
 #!/usr/bin/env python
```

Comment:
pretty sure this shebang has no effect if it's not at the beginning of the file, so it can probably be removed